src/course/handbook/project-presentations.md

```diff
@@ -0,0 +1,44 @@
+# Project Presentations
+
+Your timetable includes 45 mins for your team to write your presentation and discuss how you are going to present your project together.
+
+**Remember: We want to hear from everyone during your presentation!**
+
+Each team then has 12 minutes to present, with 5 additional minutes for questions. This means each person has about 3 minutes of speaking time. We will be strict with you on timing, so make sure you use your time effectively.
+
+### Structure
+
+1. Show your process:
+   - Talk through the Why, What, How of your project
+   - How did your team split up the work?
+   - How did your team choose to pair? How often did you swap pairs?
+2. Briefly demo your project and explain how it conforms to your brief i.e. the user stories and acceptance criteria
+3. Show your project board and (from Week 2) report on estimated vs actual velocity
```

Comment:
Maybe something about project roles here too. And is there anything else that's come up from the project management slide decks?